components/sysutils/ntfsprogs/Makefile

```diff
@@ -13,20 +13,21 @@
 # Copyright 2014 Alexander Pyhalov.  All rights reserved.
 # Copyright 2016 Adam Stevko.  All rights reserved.
 # Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+# Copyright 2021 Jean-Pierre André. All rights reserved.
 #
 
 BUILD_BITS=32
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= ntfs-3g_ntfsprogs
-IPS_COMPONENT_VERSION= 2017.3.23
-COMPONENT_VERSION= $(IPS_COMPONENT_VERSION)AR.5
+IPS_COMPONENT_VERSION= 2021.8.22
+COMPONENT_VERSION= $(IPS_COMPONENT_VERSION)
```

Comment:
IPS_COMPONENT_VERSION only needs to be explicitly set if the COMPONENT_VERSION is not a valid version according to IPS (eg. containing non-numerical characters or leading zeros).